internal/server/v1/firehose/actions.go

```diff
@@ -64,14 +79,28 @@ func (api *firehoseAPI) handleScale(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *firehoseAPI) handleStart(w http.ResponseWriter, r *http.Request) {
-	var reqBody struct{}
+	var reqBody struct {
+		StopTime *time.Time `json:"stop_time,omitempty"`
```

Comment:
do we need `StopTime` here? adding it here will make it exposed in the request where user can set it via body.

Reply:
if we don't need anything from body we can just remove the process of reading body

Reply:
Yes we would need this struct to store stoptime and pass this in request.

We shall remove the reading of request body (line 85).

Reply:
Ok, will make change in some time.